services/self-service/src/main/java/com/epam/dlab/backendapi/service/ComputationalService.java

```diff
@@ -65,7 +65,7 @@ boolean createDataEngineService(UserInfo userInfo, String resourceName, Computat
     void startSparkCluster(UserInfo userInfo, String exploratoryName, String computationalName, String project, String auditInfo);
 
     void updateSparkClusterConfig(UserInfo userInfo, String project, String exploratoryName, String computationalName,
-                                  List<ClusterConfig> config);
+                                  List<ClusterConfig> config, String auditMessage);
```

Comment:
Please rename parameter **auditMessage** -> **auditInfo**. Because we use auditInfo everywhere

Reply:
Variables names are renamed